_data/toc.yaml

```diff
@@ -253,8 +253,6 @@ reference:
           title: docker buildx imagetools inspect
         - path: /engine/reference/commandline/buildx_inspect/
           title: docker buildx inspect
-        - path: /engine/reference/commandline/buildx_install/
-          title: docker buildx install
```

Comment:
Needs to manually remove toc and files for hidden commands. We should generate TOC and avoid stub files in the future imo.